android/src/main/java/com/brentvatne/common/react/VideoEventEmitter.java

```diff
@@ -242,8 +243,14 @@ public void load(double duration, double currentPosition, int videoWidth, int vi
         load( duration,  currentPosition,  videoWidth,  videoHeight, waAudioTracks,  waTextTracks,  waVideoTracks, trackId);
     }
 
+<<<<<<<< HEAD:android/src/oldarch/java/com/brentvatne/exoplayer/VideoEventEmitter.java
```

Comment:
conflict was pushed here

Reply:
Yep, I think I've removed that merge conflicts.